src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/custom.py

```diff
@@ -126,6 +126,16 @@ def create(cmd, client, resource_group_name, resource_name, kind, description=No
 
 
 def get_bot(cmd, client, resource_group_name, resource_name, bot_json=None):
+    """Retrieves the bot's application's application settings. If called with '--msbot' flag, the operation outputs
```

Comment:
Please remove this. Command help should be placed in the `help.py` file for your module.

Reply:
The help for `az bot show` definitely needs to be updated, this docstring is for easily maintainable code (not for being shown to the user 😄). With this in mind, should I still remove it? 

I've added updating the help as a WIP item in this PR. Thanks!